riotfile.py

```diff
@@ -1732,7 +1732,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             pkgs={
                 "graphene": ["~=3.0.0", latest],
                 "pytest-asyncio": "==0.21.1",
-                "graphql-relay": "~=3.1.5",
+                "graphql-relay": latest,
```

Comment:
Should we be specifying an explicit latest version instead of the `latest` alias?

Reply:
Unless there's a maximum version that the integration supports, latest is ok.